test/jasmine/tests/histogram_test.js

```diff
@@ -463,6 +463,10 @@ describe('Test histogram', function() {
     describe('plot / restyle', function() {
         var gd;
 
+        beforeAll(function() {
+            jasmine.addMatchers(customMatchers);
```

Comment:
(note-to-self comment) I'm getting tired of these `jasmine.addMatchers`. I'll try to find a way to add our custom matchers _globally_ once and for all.

Reply:
❤️ 🎉 🥇 👏 💯

Reply:
global addMatchers, and turning `negateIf` into a method (which I could only see how to do globally) ->https://github.com/plotly/plotly.js/pull/1999/commits/0729921645f88839df29fff32097f90a60a42143 hopefully it works right on circle 🙏